README.md

```diff
@@ -14,6 +14,7 @@ A filesystem-like `ContentsManager` backend for Jupyter. This library allows you
 
 ```bash
 pip install jupyter-fs
+jupyter labextension install jupyter-fs
```

Comment:
that definitely shouldn't be necessary for a regular install. If `labextension install` really is required, that's a bug and we should fix it.

The exception is for a dev install:

```
pip install -e .[dev]
jupyter labextension link .
```

Reply:
@telamonian forgive a stupid question, but do you mean:
  * the lab extension should be installed by pip and/or conda when jupyter-fs is installed (hence no separate `labextension install` step needed), or
  * the lab extension should not always be required in the first place (e.g. because jupyter-fs provides some functionality unrelated to jupyter lab), so it should be in an 'optional' section?

I guess the first option (but let me know if you meant something totally different!).

I ask because I had wondered about simplifying the user installation of nbcelltests by having pip and conda install the labextension, so I'd be interested to hear what you propose/do here.

(nbcelltests is actually useful - or should be! - without the lab extension, so currently we have `pip install nbcelltests` followed by `jupyter labextension install jupyterlab_celltests`. But instead it could just be `pip install nbcelltests[jupyterlab]` or something like that for people who want to use it with lab.)

Reply:
The first one. [jupyterlab-git](https://github.com/jupyterlab/jupyterlab-git) and [nbdime](https://github.com/jupyter/nbdime) are both existing example of python packages that also include a pre-bundled labextension. Neither is perfect, but they're both functional